src/python/datastore/data_handler.py

```diff
@@ -301,7 +301,7 @@ def get_fuzzer_display(testcase):
       fully_qualified_name=fuzz_target.fully_qualified_name())
 
 
-def get_arguments(testcase):
+def get_arguments(testcase, strip_target=True):
```

Comment:
we can just remove modifying this function, can just use get_fuzzer_display above.

Reply:
Done!